packages/ember-metal/lib/chains.js

```diff
@@ -119,7 +119,8 @@ function lazyGet(obj, key) {
   }
 
   // if a CP only return cached value
-  var desc = meta && meta.descs[key];
+  var possibleDesc = obj[key];
+  var desc = (possibleDesc !== null && typeof possibleDesc === 'object' && possibleDesc.isDescriptor) ? possibleDesc : undefined;
```

Comment:
I would like a bikeshedding around the fastest way to check if it's a descriptor @stefanpenner @krisselden